src/functions/instance/state/describeTableCache.js

```diff
@@ -0,0 +1,23 @@
+const descriptionCache = new Map(JSON.parse(sessionStorage.getItem('descriptionCache')) || []);
+
+const getTableKey = (schema, table) => `${schema}/${table}`;
+
+const getTableDescriptionFromCache = (schema, table) => {
+	const tableKey = getTableKey(schema, table);
+	if (!descriptionCache.has(tableKey)) return null;
+	return descriptionCache.get(tableKey);
```

Comment:
Simplify, `.get()` will return `undefined` if the thing isn't in the Map so you can use `?? null` to eliminate the `.has` check.
```suggestion
	return descriptionCache.get(tableKey) ?? null;
```

Reply:
Or just do a .get and let it return `undefined` (`undefined` is a good value to describe an undefined value).